integration-tests/main/src/main/java/io/quarkus/it/websocket/WebSocketOpenEndpoint.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.it.websocket;
+
+import java.io.IOException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+import javax.websocket.server.ServerEndpoint;
+
+@ServerEndpoint("/wsopen")
+@ApplicationScoped
+public class WebSocketOpenEndpoint {
+
+    public static String[] messages = { "23.0", "1.0", "99.0", "10.0", "10.0", "10.0", "10.0", "10.0" };
+
+    @OnOpen
+    public void onOpen(Session session) throws IOException {
+        for (String i : messages) {
```

Comment:
Because the registration of the handler happens after the opening, we may miss messages. That's expected.
We could modify the test to receive a first ping before sending the messages.